scipy/stats/_distn_infrastructure.py

```diff
@@ -2378,7 +2394,10 @@ def expect(self, func=None, args=(), loc=0, scale=1, lb=None, ub=None,
         Notes
         -----
         The integration behavior of this function is inherited from
-        `integrate.quad`.
+        `integrate.quad`. Neither this function nor `integrate.quad` can 
```

Comment:
Can you write scipy.integrate.quad (this time single backticks,then the function will be linked)

Reply:
@chrisb83, do we also want the cauchy to have the full path in the docstring?

Reply:
If you use single backticks, the function is linked, I think it searches in the same module by default, thats why we need to put scipy so that the function can be found in the other module.

The cauchy part is simply code that you can print as monospace using double-backticks.

Reply:
I ran: `python setup.py build_sphinx` and looked at the html doc in `$PATH_PREFIX/scipy/doc/build/html/generated/scipy.stats.rv_continuous.html#scipy.stats.rv_continuous` I missed that the `integrate.quad` method is a separate module, Thanks!